docs/input/support/index.cshtml

```diff
@@ -0,0 +1,65 @@
+NoSidebar: true
+NoChildPages: true
+Title: Support The Project ❤
+---
+
+<div class="row">
+    <div class="col-md-6 mb-5">
+        <h1>Sponsor Development</h1>
+
+        <div class="text-center my-3">
+            <i class="fad fa-heart fa-5x icon-red"></i>
+        </div>
+        <p>
+            Tasty isn't developed by a big, faceless corporation or backed by mountains of venture captital.<br>
+            It's mostly <a href="https://github.com/biohazard999" target="_blank">Manuel</a> and other awesome contributors spending lots of nights and weekends as a labor of love.
+        </p>
+        <p>
+            Development and support of Tasty has been taken of real quick and there are no plans to slow down.<br>
```

Comment:
Rephrase. You mean 'take of' like ✈?
Perhaps:
Development and support of Tasty took off quickly and there are no plans to slow down.

Reply:
Fixed